packages/nextra/src/loader.ts

```diff
@@ -243,43 +257,38 @@ async function loader(
   const cssImport = OFFICIAL_THEMES.includes(theme)
     ? `import '${theme}/style.css'`
     : ''
-  const finalResult = transform
-    ? await transform(result, { route: pageNextRoute })
-    : result
   const stringifiedPageOpts = JSON.stringify(pageOpts)
+  const pageOptsChecksum = IS_PRODUCTION
+    ? "''"
+    : JSON.stringify(hashFnv32a(stringifiedPageOpts))
+  const dynamicMetaModules = dynamicMetaItems
+    .map(
+      descriptor =>
+        `[import(${JSON.stringify(descriptor.metaFilePath)}), ${JSON.stringify(
+          descriptor
+        )}]`
+    )
+    .join(',')
 
   return `import { setupNextraPage } from 'nextra/setup-page'
 import __nextra_layout from '${layout}'
 ${themeConfigImport}
 ${katexCssImport}
 ${cssImport}
-
-${finalResult.replace(
-  'export default MDXContent;',
-  "export { default } from 'nextra/layout'"
-)}
+${finalResult}
 
 setupNextraPage({
-  Content: MDXContent,
+  MDXContent,
   nextraLayout: __nextra_layout,
   hot: module.hot,
   pageOpts: ${stringifiedPageOpts},
   themeConfig: ${themeConfigImport ? '__nextra_themeConfig' : 'null'},
   pageNextRoute: ${JSON.stringify(pageNextRoute)},
-  pageOptsChecksum: ${
-    IS_PRODUCTION
-      ? 'undefined'
-      : JSON.stringify(hashFnv32a(stringifiedPageOpts))
-  },
-  dynamicMetaModules: typeof window === 'undefined' ? [${dynamicMetaItems
-    .map(
-      descriptor =>
-        `[import(${JSON.stringify(descriptor.metaFilePath)}), ${JSON.stringify(
-          descriptor
-        )}]`
-    )
-    .join(',')}] : []
-})`
+  pageOptsChecksum: ${pageOptsChecksum},
+  dynamicMetaModules: typeof window === 'undefined' ? [${dynamicMetaModules}] : []
+})
+
+export { default } from 'nextra/layout'`
```

Comment:
I think `nextra/layout` we can move directly to `nextra/setup-page` to have less file size